fujiji-server/src/repositories/boost.js

```diff
@@ -15,10 +15,24 @@ async function createBoost(listingID, packageid, score) {
       type: Sequelize.INSERT,
     },
   );
+  console.log(result);
```

Comment:
I don't think we need this `console.log`, right?

Reply:
yes, will remove this